.evergreen/auth_aws/aws_e2e_ecs.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Validate that MONGODB-AWS auth works from ECS temporary credentials.
+ */
+load("lib/aws_e2e_lib.js");
+
+(function() {
+   'use strict';
+
+   assert.eq(typeof mongo_binaries != 'undefined', true, "mongo_binaries must be set");
```

Comment:
FYI, You set these via the command line if you wants

./mongo --eval "mongo_binaries='foo';project_dir='bar'" aws_e2e_ecs.js 

Some of our tests use this approach.